x11-themes/mate-backgrounds/mate-backgrounds-1.27.1.ebuild

```diff
@@ -5,8 +5,9 @@ EAPI=8
 
 inherit mate
 
-if [[ ${PV} != 9999 ]]; then
-	KEYWORDS="~amd64 ~arm ~arm64 ~loong ~riscv ~x86"
+MINOR=$((ver_cut 2) % 2)
```

Comment:
This is still missing a $.

Reply:
Sorry about that, it's fixed in the clean up commit.